comp/core/workloadmeta/collectors/internal/kubeapiserver/pod.go

```diff
@@ -73,6 +73,26 @@ func newPodParser(annotationsExclude []string) (objectParser, error) {
 	return podParser{annotationsFilter: filters}, nil
 }
 
+func convertResource(c corev1.ResourceList) map[string]string {
+	res := make(map[string]string, len(c))
+	for k, v := range c {
+		res[string(k)] = v.String()
+	}
+	return res
+}
+
+func containersToOrchestratorContainers(cont []corev1.Container) []workloadmeta.OrchestratorContainer {
+	containers := make([]workloadmeta.OrchestratorContainer, 0, len(cont))
```

Comment:
For now, does it make sense to short-circuit this function and have it return an empty list or nil if `cfg.GetBool("autoscaling.workload.enabled")` is not true?

Reply:
I think it would absolutely make sense

Reply:
Done in the last commit